### PR TITLE
Adding Tooltip component

### PR DESCRIPTION
**Pull request by ryanml**
Addresses #106

### CODE REVIEW

File: src/features/rewards/tooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import {
+  StyledWrapper,
+  StyledTooltip,
+  StyledTooltipText,
+  StyledPointer
+} from './style'
+
+export interface Props {
+  id?: string
+  text?: string
```
Comment (NejcZdovc) [inline src/features/rewards/tooltip/index.tsx]: I think this should be `React.ReactNode` and not string

Comment (ryanml) [reply src/features/rewards/tooltip/index.tsx]: done

**Comment by NejcZdovc** [comment]
@ryanml please rebase

File: src/features/rewards/tooltip/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import {
+  StyledWrapper,
+  StyledTooltip,
+  StyledTooltipText,
+  StyledPointer
+} from './style'
+
+export interface Props {
+  id?: string
+  content?: React.ReactNode
```
Comment (NejcZdovc) [inline src/features/rewards/tooltip/index.tsx]: I think that content should be required

File: src/features/rewards/tooltip/style.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+*/
+
+import styled from 'styled-components'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  z-index: 2;
+  position: relative;
+`
+
+export const StyledTooltip = styled<{}, 'div'>('div')`
```
Comment (NejcZdovc) [inline src/features/rewards/tooltip/style.ts]: Please use styledProps

File: src/features/rewards/tooltip/style.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  top: calc(100% + 10px);
+  transform: translateX(-50%);
+  white-space: nowrap;
+  position: absolute;
+  background: #0C0D21;
+  text-align: center;
+  padding: 10px 5px;
+  border-radius: 3px;
+  box-shadow: 1px 1px 5px 0 rgba(34, 35, 38, 0.43);
+  display: ${(p: {displayed: boolean}) => p.displayed ? 'inline-block' : 'none'};
```
Comment (NejcZdovc) [inline src/features/rewards/tooltip/style.ts]: `(p: {displayed: boolean})` -> `p`

**Pull request merged**